src/controllers/objecting.entity.name.controller.ts

```diff
@@ -1,27 +1,66 @@
 import { NextFunction, Request, Response } from "express";
+import { check, ValidationError, validationResult } from "express-validator";
+import { ErrorMessages } from "../model/error.messages";
+import { createGovUkErrorData, GovUkErrorData } from "../model/govuk.error.data";
 import { OBJECTIONS_COMPANY_NUMBER } from "../model/page.urls";
 import { addObjectionCreateToObjectionSession } from "../services/objection.session.service";
 import { ObjectionCreate } from "../modules/sdk/objections";
 import { Session } from "ch-node-session-handler";
+import { Templates } from "../model/template.paths";
+
+const FULL_NAME_FIELD: string = "fullName";
```

Comment:
when declaring string literal constants you can leave off the : string 
ie. x = "some string", you don't need : string 
x: string = getMessage() you would need it.

Reply:
it's ok as it is, just a best practice thing